pkg/snapshot/rule.go

```diff
@@ -51,9 +51,15 @@ func validateSnapRules(snap *crdv1.VolumeSnapshot) error {
 	return nil
 }
 
+func setKind(snap *crdv1.VolumeSnapshot) {
+	snap.Kind = "VolumeSnapshot"
```

Comment:
VolumeSnapshot objects don't have kind set by default?

Reply:
From the commit message
```
It doesn't seem to be set always when being passed in from the snapshot
controller.
```